src/qml/DigitizingToolbar.qml

```diff
@@ -19,6 +19,7 @@ VisibilityFadingRow {
   spacing: 4
 
   signal confirm
+  signal cancel
```

Comment:
Should we follow the qdoc syntax (https://doc.qt.io/qt-5/qtwritingstyle-qml.html#signals-and-handlers-documentation) for these?
I.e.

>     This signal is emitted when [...] . The corresponding handler is \c onCancel.